### PR TITLE
Replace env with params and add additional

### PR DESCRIPTION
**Pull request by jcpowermac**
- Added env vars for dev and stage project name
- Added Dev OpenShift uri
- Added Checkout path restriction

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -6,10 +6,24 @@ pipeline {
     environment {
         IMAGE_STREAM_TAG = "latest"
         REGISTRY = "docker-registry.engineering.redhat.com"
-        PROJECT = "lifecycle"
+        DEV_PROJECT = "lifecycle"
+        STAGE_PROJECT = "lifecycle"
         STAGE_URI = "insecure://openshift-ait.e2e.bos.redhat.com:8443"
+        DEV_URI = "insecure://osemaster.sbu.lab.eng.bos.redhat.com:8443"
```
Comment (aweiteka) [inline Jenkinsfile]: Should we put all of these into env.VAR that is passed in from the OpenShift BC?

File: Jenkinsfile [hunk 2]
```diff
@@ -6,10 +6,24 @@ pipeline {
...
     }
     stages {
+        stage('Checkout') {
+            steps {
+				checkout([$class: 'GitSCM',
```
Comment (aweiteka) [inline Jenkinsfile]: nit: indent

File: Jenkinsfile (hunk 1 above)
Comment (aweiteka) [reply Jenkinsfile]: Or maybe these will be overridden by those in the BC

Comment (aweiteka) [reply Jenkinsfile]: AFICT, openshift BC env vars are mapped as both params.KEY and regular environment vars.

Comment (jcpowermac) [reply Jenkinsfile]: @aweiteka let me test this

**Pull request merged**